### PR TITLE
Cross reference NativeAOT related docs

### PR DESCRIPTION
**Pull request by kant2002**
I will help reader find other NativeAOT pieces faster

### CODE REVIEW

File: docs/using-nativeaot/README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Using Native AOT
 
+- [Building](../workflow/building/coreclr/nativeaot.md)
```
Comment (jkotas) [inline docs/using-nativeaot/README.md:3]: People will get to this doc by clicking on "how to debug, optimize and troubleshoot applications published using the native AOT toolchain" in the readme.

How to build the repo is not relevant for that.

**Comment by kant2002** [comment]
Because NativeAot is folder, I found that location more often. And original
location with build instructions I sometimes even cannot find at all.
That’s actually the reason why I decide to provide hints to other locations
with related information

**Comment by jkotas** [comment]
> Because NativeAot is folder, I found that location more often.

The file you are referencing has nativeaot in the name as well...

We have intentional split between "how to build the repo" and "how to use the toolchain" documentation. When we had these types of documentations intermixed in the past, we had seen number of cases where people started going through the pain of building the repo to try it out, but all they actually needed to do was to just add the package reference.

**Comment by kant2002** [comment]
Now I understand why I see that gap. I may have to think of there is a way
how his gap can be crossed without increasing support costs for you guys.

**Comment by jkotas** [comment]
Closing this for now

**Pull request closed without merging**